sensorlib/src/objects/DetectedObject.py

```diff
@@ -15,28 +15,32 @@
 from util.CarlaUtils import CarlaUtils
 
 
+
 @dataclass(frozen=True)
 class DetectedObject:
     """Wrapper class for carla.Actor which are detected by the sensor."""
-    carla_actor: carla.Actor
-    id: int
-    object_type: str
-    timestamp: int
-    bounding_box_in_world_coordinate_frame: List[np.ndarray]
+    objectId: int
+    type: str
     position: np.ndarray
     velocity: np.ndarray
     rotation: np.ndarray
-    angular_velocity: np.ndarray
-    position_covariance: np.ndarray
-    orientation_covariance: np.ndarray
-    velocity_covariance: np.ndarray
-    angular_velocity_covariance: np.ndarray
+    angularVelocity: np.ndarray
+    positionCovariance: np.ndarray
+    velocityCovariance: np.ndarray
+    orientationCovariance: np.ndarray
+    angularVelocityCovariance: np.ndarray  
     confidence: float
+    projString: str
+    size: np.ndarray
+    timestamp: int
+    sensorId: str
+    carla_actor: carla.Actor  
+    bounding_box_in_world_coordinate_frame: List[np.ndarray]
```

Comment:
Let's search in the sensorlib directory for every occurences of these name changes and change them. On top of the id name change, I see changes required for object_type for example. Please double check